src/parser.ts

```diff
@@ -16,7 +16,14 @@ function normalizeField(field: string) {
 const stateSet = {
   main: {
     [TOKEN_TYPES.OPEN_BLOCK]: { match: '[', push: 'block' },
-    [TOKEN_TYPES.INCLUDE]: { match: /@include+\s.*/, lineBreak: true },
+    [TOKEN_TYPES.SET]: [
+      { match: /@set+\s+\w=.*/, lineBreak: true, value: (value: string) => value.replace(/@set/, '@SET') },
+      { match: /@SET+\s+\w=.*/, lineBreak: true },
```

Comment:
I think we're good for set, the rest is linter.

Reply:
Once we have mixed case I think this can be merged, currently I believe it won't accept `@Set` which seems quite a common one but also all the other mixed case options.

Reply:
It does accepts @Set as well, I will make a use case with the common cases.